Automattic/wp-calypso
# Domains: Cleanup `domains/cctlds` feature flags and Lodash usage

#### Changes proposed in this Pull Request

This PR cleans up the always enabled `domains/cctlds` flags, and cleans up some related Lodash usage.

These feature flags are always enabled, so they can just be removed. The Lodash usage there is also pretty straightforward to migrate.

#### Testing instructions

* Add a `.fr` domain to your cart and go to `/checkout/:domain`.
* Verify you can still see the FR-specific form at the bottom (the "Almost done! We need some extra details to register your .fr domain." text and the form fields below).
